src/Config/ValueProvider/TestFrameworkConfigPathProvider.php

```diff
@@ -77,8 +80,8 @@ public function get(InputInterface $input, OutputInterface $output, array $dirsI
         try {
             $this->testFrameworkConfigLocator->locate($testFramework);
 
-            return null;
```

Comment:
is it a correct change? I think `?string` is expected

Reply:
https://github.com/infection/infection/pull/865#discussion_r351839497